packages/middleware-sdk-glacier/src/add-glacier-api-version.spec.ts

```diff
@@ -3,15 +3,25 @@ import { HttpRequest } from "@aws-sdk/protocol-http";
 
 describe("addGlacierApiVersion", () => {
   const mockNextHandler = jest.fn();
+  const unusedDep = jest.fn();
 
   beforeEach(() => {
     jest.clearAllMocks();
   });
 
+  const config = {
+    apiVersion: "1970-01-01",
+    bodyChecksumGenerator: unusedDep,
+    runtime: "node",
```

Comment:
nit: Is `runtime` used here?

Reply:
no. leftover from before. fixed in 8054b552a